uc-crux-llvm/src/UCCrux/LLVM/Main/Config/FromEnv.hs

```diff
@@ -73,13 +114,43 @@ processUCCruxLLVMOptions (initCOpts, initUCOpts) =
             },
           ucLLVMOptions initUCOpts
         )
-    pure (appCtx, finalCOpts, initUCOpts {ucLLVMOptions = finalLLOpts})
 
-crashEquivalenceDoc :: Text
-crashEquivalenceDoc = "Check crash-equivalence with another LLVM bitcode module"
+    let topConf =
+          Config.TopLevelConfig
+            { Config.ucLLVMOptions = finalLLOpts,
+              Config.runConfig =
+                case entries of
+                  Just ents -> Config.RunOn ents
+                  Nothing ->
+                    if doExplore initUCOpts
+                    then
+                      Config.Explore
+                        (ExConfig.ExploreConfig
+                          { ExConfig.exploreAgain = reExplore initUCOpts,
+                            ExConfig.exploreBudget = exploreBudget initUCOpts,
+                            ExConfig.exploreTimeout = exploreTimeout initUCOpts,
+                            ExConfig.exploreParallel = exploreParallel initUCOpts,
+                            ExConfig.exploreSkipFunctions = skipFunctions initUCOpts
+                          })
+                    else
+                      Config.CrashEquivalence
+                        (EqConfig.EquivalenceConfig
+                          { EqConfig.equivOrOrder =
+                              if crashEquivalence initUCOpts
+                              then EqConfig.Equivalence
+                              else EqConfig.Order,
+                            EqConfig.equivModule = crashOrder initUCOpts,
+                            EqConfig.equivEntryPoints = entryPoints initUCOpts
+                          })
+            }
 
-crashEquivalenceStrictDoc :: Text
-crashEquivalenceStrictDoc = "Check for strict crash equivalence"
+    return (appCtx, finalCOpts, topConf)
+
+crashOrderDoc :: Text
+crashOrderDoc = "Check crash-ordering with another LLVM bitcode module"
```

Comment:
And I apologize for a more generic comment on things without reviewing the overall beyond the "window" provided by just the PR changes, but I just wanted to make sure this was covered somewhere and this looked like a place that a user might see and need to know this information.

Reply:
> And I apologize for a more generic comment on things without reviewing the overall beyond the "window" provided by just the PR changes, but I just wanted to make sure this was covered somewhere and this looked like a place that a user might see and need to know this information.

Not at all! This is all IR&D code so it can use a thorough review.